docs/docsite/rst/network/user_guide/platform_eos.rst

```diff
@@ -0,0 +1,129 @@
+.. _eos_platform_options:
+
+***************************************
+EOS Platform Options
+***************************************
+
+Arista EOS supports multiple connections. This page offers details on how each connection works in Ansible 2.5 and how to use it. 
+
+.. contents:: Topics
+
+Connections Available
+================================================================================
+
++---------------------------+-----------------------------------------------+-----------------------------------------+
+|..                         | CLI                                           | eAPI                                    |
++===========================+===============================================+=========================================+
+| **Protocol**              |  SSH                                          | HTTP(S)                                 |
++---------------------------+-----------------------------------------------+-----------------------------------------+
+| | **Credentials**         | | uses SSH keys / SSH-agent if present        | | uses HTTPS certificates if present    |
+| |                         | | accepts ``-u myuser -k`` if using password  | |                                       |
++---------------------------+-----------------------------------------------+-----------------------------------------+
+| **Indirect Access**       | via a bastion (jump host)                     | via a web proxy                         |
++---------------------------+-----------------------------------------------+-----------------------------------------+
+| | **Connection Settings** | | ``ansible_connection: network_cli``         | | ``ansible_connection: local``         |
+| |                         | |                                             | | Requires ``transport: eapi``          |
+| |                         | |                                             | | in the ``provider`` dictionary        |
++---------------------------+-----------------------------------------------+-----------------------------------------+
+| | **Enable Mode**         | | supported - use ``ansible_become: yes``     | | supported - use ``authorize: yes``    |
+| | (Privilege Escalation)  | | with ``ansible_become_method: enable``      | | in the ``provider`` dictionary        |
++---------------------------+-----------------------------------------------+-----------------------------------------+
+| **Returned Data Format**  | ``stdout[0].``                                | ``stdout[0].messages[0].``              |
++---------------------------+-----------------------------------------------+-----------------------------------------+
```

Comment:
Related to return data format for *_command network modules to access the result in wait_for can we update something like
```
        Note - With I(wait_for) the value in C(result['stdout']) can be accessed
        using C(result), that is to access C(result['stdout'][0]) use C(result[0]) See examples.
```
at a common place in this guide?

Reply:
Thanks for the feedback. I need to think about where this would work best in the overall documentation scheme. I've created https://github.com/ansible/ansible/issues/36872 to track this work.